### PR TITLE
[ON HOLD] Feature/content translation

### PR DESCRIPTION
**Pull request by tebajanga**
## Description
Adding content translation on backend and frontend.
This feature use Wagtail Modeltranslation [https://github.com/infoportugal/wagtail-modeltranslation](https://github.com/infoportugal/wagtail-modeltranslation)

Fixes # [114](https://github.com/TakwimuAfrica/TAKWIMU/issues/114)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Screenshots
takwimu_translation_dashboard

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/translation.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from .models import TopicPage
+from modeltranslation.translator import TranslationOptions
+from modeltranslation.decorators import register
+
+@register(TopicPage)
+class TopicPageTR(TranslationOptions):
+    fields = (
+        'description',
+    )
```
Comment (DavidLemayian) [inline takwimu/translation.py]: I'm supposing we intend to register all the models?

**Comment by DavidLemayian** [comment]
@tebajanga You also have the failing tests. Must be order of urlpatterns added.

**Comment by tebajanga** [comment]
@DavidLemayian, I was able to register only Topic and ProfileSection Pages.

**Comment by DavidLemayian** [comment]
@tebajanga What issue are you having with the rest of the models?

**Comment by tebajanga** [comment]
@DavidLemayian I was getting some errors, for example when I was registering` indicator` field of **TopicPageDataIndicator** model it shows
 ```
raise FieldDoesNotExist('%s has no field named %r' % (self.object_name, field_name))
django.core.exceptions.FieldDoesNotExist: TopicPageDataIndicator has no field named 'a'
```

**Comment by kilemensi** [comment]
Archiving repo

**Pull request closed without merging**